titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/JobManagerUtil.java

```diff
@@ -148,6 +149,20 @@ public static Task attachPlacementData(Task task, Optional<TitusExecutorDetails>
         }).orElse(task);
     }
 
+    public static Task attachKubeletData(Task task, PodWrapper podWrapper) {
+        if(podWrapper.getV1Pod().getStatus() == null) { return task; }
```

Comment:
minor: formatting is odd here. There's code style config file for IntelliJ you can use in the `codequality/` folder